DEV.md

```diff
@@ -0,0 +1,51 @@
+# Getting Started
+
+Miranum Connect is powered by maven. To set up the moduls run `mvn build`.
+
+## Testing
+
+Start the example projects.
+
+## Publishing
+
+As a developer of Miragon GmbH, you can access the repos actions and publish a new release by clicking "Run Workflow" in the [release section](https://github.com/FlowSquad/miranum/actions/workflows/release.yaml).
+Please enter the new version number as release tag.
+
```

Comment:
Before or after the release you have to increase the mvn version otherwise the release to maven central will fail.

```
mvn versions:set -DnewVersion="0.1.0"
mvn versions:commit
```

And commit the file changes